ci-build/test.sh

```diff
@@ -11,9 +11,14 @@ panic() {
 # Move to project root
 cd "$(dirname "$0")/.."
 
-if [[ -z "$SKIP_FORMAT_CHECK" ]]; then
```

Comment:
Maybe it should be `SKIP_FORMAT_CHECK`?

Reply:
okay, I made dirty checking a separate flag from the individual checks.